inception-project/inception
# Feature/705 annotations in bulk

**What's in the PR**
* Feature for creating/deleting annotations for all search results 

**How to test manually**
* See Issue #705
* See if the described functionality is working

**Automatic testing**

**Documentation**
